chess/3-web-api/web-api.md

```diff
@@ -422,7 +422,9 @@ Successfully run the service unit tests that you created. They must directly cal
 
 ### Code Quality
 
-For this phase the auto grader will grade the quality of all your project's source code. The rubric used to evaluate code quality can be found here: [Rubric](../code-quality-rubric.md). You can also test your quality with the auto grader independent of a specific phase submission.
+For this phase the auto grader will grade the quality of all your project's source code.
```

Comment:
My editor wraps long lines, why make these explicit? This will make it more difficult to maintain the paragraph when changes are made.

Reply:
Heard. I turned on line wrapping by default for `*.txt` and `*.md` files in my editor, and I'll put the content all on one line.